fury/trees.py

```diff
@@ -0,0 +1,817 @@
+from fury.utils import Actor
+from fury.actor import line
+import numpy as np
+
+
+# QUADTREE IMPLEMENTATION
+class Point2d():
```

Comment:
No need to have classes for Point2D and point 3D. we already use lists or numpy array interchangeably to store vectors. It is also faster because you can store several vectors in numpy.

Reply:
I believe the problem is more of a confusion that the class name is generating. Point2D, Point3D are not just implementations of positions.